src/kakarot/accounts/library.cairo

```diff
@@ -27,6 +27,10 @@ from kakarot.constants import (
 )
 from kakarot.interfaces.interfaces import IAccount
 
+@event
+func evm_contract_deployed(evm_contract_address: felt, starknet_contract_address: felt) {
```

Comment:
@Eikix Is there more info we might need for the rpc/block-explorer?
Maybe the evm address that deployed the contract?

Reply:
I'm not sure yet, does emitting an event pose a problem to the app?

I recall @ClementWalter saying it was ruining the event parsing in his tests

Reply:
Oh, I thought it just got removed because it wasn't being used.

Reply:
Actually, there is not real problem it was only for events parsing I’m the web3.pu lib.

Reply:
We can leave it there!